### PR TITLE
SNOW-141055: Add test run scripts

### PR DESCRIPTION
**Pull request by sfc-gh-dprzybysz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,43 +37,51 @@ at <a href="https://docs.snowflake.net/manuals/user-guide/nodejs-driver.html">No
 Test
 ======================================================================
 
-Prepare for Test
+Prepare for Tests
```
Comment (sfc-gh-pfus) [inline README.md:40]: To be perfectly correct, I would write it as `tests`.

**Comment by sfc-gh-dprzybysz** [comment]
I have read the CLA Document and I hereby sign the CLA

File: README.md [hunk 2]
```diff
@@ -37,43 +37,51 @@ at <a href="https://docs.snowflake.net/manuals/user-guide/nodejs-driver.html">No
...
-    "SNOWFLAKE_TEST_PASSWORD":  "&lt;your_password&gt;",
-    "SNOWFLAKE_TEST_ACCOUNT":   "&lt;your_account&gt;",
-    "SNOWFLAKE_TEST_WAREHOUSE": "&lt;your_warehouse&gt;",
-    "SNOWFLAKE_TEST_DATABASE":  "&lt;your_database&gt;",
-    "SNOWFLAKE_TEST_SCHEMA":    "&lt;your_schema&gt;",
-    "SNOWFLAKE_TEST_ROLE":      "&lt;your_role&gt;"
-  }
-}
-</code></pre>
+Specify env variables:
```
Comment (sfc-gh-pfus) [inline README.md:43]: I noticed, that this file is indeed used in some scripts. Are you sure, that we want to get rid of it in this readme?

File: README.md [hunk 3]
```diff
@@ -37,43 +37,51 @@ at <a href="https://docs.snowflake.net/manuals/user-guide/nodejs-driver.html">No
...
+```
+export SNOWFLAKE_TEST_USER=<your_user>
+export SNOWFLAKE_TEST_PASSWORD=<your_password>
+export SNOWFLAKE_TEST_ACCOUNT=<your_account>
+export SNOWFLAKE_TEST_WAREHOUSE=<your_warehouse>
+export SNOWFLAKE_TEST_DATABASE=<your_database>
+export SNOWFLAKE_TEST_SCHEMA=<your_schema>
+export SNOWFLAKE_TEST_PROTOCOL=<your_snowflake_protocol>
+export SNOWFLAKE_TEST_HOST=<your_snowflake_host>
+export SNOWFLAKE_TEST_PORT=<your_snowflake_port>
```
Comment (sfc-gh-pfus) [inline README.md:54]: Are there some default values? Maybe we should somehow mention it here, that these values are not required?

File: README.md (hunk 2 above)
Comment (sfc-gh-dprzybysz) [reply README.md:43]: parameters.json is only used in CI tests. It's not necessary to have this file created for local tests.

Comment (sfc-gh-igarish) [reply README.md:43]: You can use this file if environment variables are not set. So better to keep it.

File: README.md (hunk 3 above)
Comment (sfc-gh-igarish) [reply README.md:54]: There are no defaults.

**Comment by sfc-gh-igarish** [comment]
@sfc-gh-dprzybysz Could you please create an issue? If you think there is no code change and no need for an issue that's also fine. But as a best practice each PR should link to an issue.

**Review by sfc-gh-igarish — APPROVED** [review]
LGTM. Please make sure formatting is good.

File: README.md (hunk 2 above)
Comment (sfc-gh-dprzybysz) [reply README.md:43]: Parameters.json are only used on CI. For running tests locally they are not used. @sfc-gh-igarish do you think we should keep it in readme?

**Pull request merged**